demo/addons/fmod/nodes/FmodEventEmitter.gd

```diff
@@ -50,6 +53,7 @@ func pause() -> void:
 func _unpause() -> void:
 	if event_id != UNDEFINED:
 		Fmod.set_event_paused(event_id, false)
+	is_paused = false
```

Comment:
Quick thought we just had with Piertho. 
It's not a good idea to have a `is_paused` property in that node as it already exist inside the FMOD API (so 2 times the same data).
In itself, it's not that bad (Godot does that a lot between nodes and internal server representation). But here the issue is that the `paused` statut can be change outside this node.
The 2 main causes would be:
- Set the pause from another script
- `Fmod.pause_all_event() `is called

So it would be better if you only use `get/set_event_pause()`. Make it a property with custom setter/getter if you think it's more convenient.

Reply:
pushed!